components/AboutAuthor/AboutAuthor.js

```diff
@@ -0,0 +1,42 @@
+import Image from "next/image";
+
+import kid from "../../public/kid.png";
+import styles from "./AboutAuthor.module.css";
+
+// mock data/api
+const fetchMockData = (slug) => ({
+  id: 1234,
+  authorAvatar: kid,
+  authorUsername: "trinh",
+  authorHandle: "@trinhandandy",
```

Comment:
would either of these just be their blog name or are they something separate? in the db right now there's displayName which is a full name `Firstname Lastname` and blogName, which is more of a handle/blog slug `TestBlog`

Reply:
Yeah i just merged all the recent APIs and am working on retrieving blogs now that I have an idea how you guys are working with the DB.
I can update this authorCard to fit our schema.